samples/dotnet/kernel-syntax-examples/Example12_Planning.cs

```diff
@@ -120,6 +122,59 @@ private static async Task BookSamplesAsync()
         await ExecutePlanAsync(kernel, planner, originalPlan);
     }
 
+    private static async Task MemorySampleAsync()
+    {
+        Console.WriteLine("======== Planning - Create and Execute Plan using Memory ========");
+
+        var kernel = new KernelBuilder()
+            .WithLogger(ConsoleLogger.Log)
+            .Configure(
+                config =>
+                {
+                    config.AddAzureOpenAICompletionBackend(
+                        Env.Var("AZURE_OPENAI_DEPLOYMENT_LABEL"),
+                        Env.Var("AZURE_OPENAI_DEPLOYMENT_NAME"),
+                        Env.Var("AZURE_OPENAI_ENDPOINT"),
+                        Env.Var("AZURE_OPENAI_KEY"));
+
+                    config.AddAzureOpenAIEmbeddingsBackend(
+                        Env.Var("AZURE_OPENAI_EMBEDDINGS_DEPLOYMENT_LABEL"),
+                        Env.Var("AZURE_OPENAI_EMBEDDINGS_DEPLOYMENT_NAME"),
+                        Env.Var("AZURE_OPENAI_EMBEDDINGS_ENDPOINT"),
+                        Env.Var("AZURE_OPENAI_EMBEDDINGS_KEY"));
+                })
+            .WithMemoryStorage(new VolatileMemoryStore())
+            .Build();
+
+        // Load native skill into the kernel skill collection, sharing its functions with prompt templates
+        var planner = kernel.ImportSkill(new PlannerSkill(kernel), "planning");
+
+        string folder = RepoFiles.SampleSkillsPath();
+        kernel.ImportSemanticSkillFromDirectory(folder, "SummarizeSkill");
+        kernel.ImportSemanticSkillFromDirectory(folder, "WriterSkill");
+        kernel.ImportSemanticSkillFromDirectory(folder, "CalendarSkill");
+        kernel.ImportSemanticSkillFromDirectory(folder, "ChatSkill");
+        kernel.ImportSemanticSkillFromDirectory(folder, "ChildrensBookSkill");
+        kernel.ImportSemanticSkillFromDirectory(folder, "ClassificationSkill");
+        kernel.ImportSemanticSkillFromDirectory(folder, "CodingSkill");
```

Comment:
Do we really need all these skills for this example, or is part of the point having skills that are not necessary? If so, let's add a comment stating this.

Reply:
Yes that's the point. Will do.